src/server/server.module.ts

```diff
@@ -21,7 +21,7 @@ import { AuthenticationApiModule } from '../modules/authentication/authenticatio
 import { PersonenKontextApiModule } from '../modules/personenkontext/personenkontext-api.module.js';
 import { ServiceProviderApiModule } from '../modules/service-provider/service-provider-api.module.js';
 import { SessionAccessTokenMiddleware } from '../modules/authentication/services/session-access-token.middleware.js';
-import { createClient, RedisClientType } from 'redis';
+import { createClient, createCluster, RedisClientType, RedisClusterType } from 'redis';
```

Comment:
@kristoff-kiefer could you please check if changes in https://github.com/dBildungsplattform/dbildungs-iam-server/blob/main/src/modules/health/redis.health-indicator.ts are also necessary?